Source/Charts/Components/ChartLimitLine.swift

```diff
@@ -44,6 +44,11 @@ open class ChartLimitLine: ComponentBase
     @objc open var imageInsect = UIEdgeInsets.zero
     @objc open var imageSize = CGSize.zero
     @objc open var imageStickyLength = Double(0)
+    @objc open var imageTint: UIColor? = nil
+    /// Radial Gradient around limit line icons
+    @objc open var radialGradientColors: CFArray = [CGColor]() as CFArray
```

Comment:
should we use the old default value here? `[UIColor.white.withAlphaComponent(0).cgColor, UIColor.white.cgColor]`